### PR TITLE
Fix GELU-related docstring formatting

### PR DESCRIPTION
**Pull request by MysticalMusings**
The docstring about GELU seems formatted incorrectly. The original docstring about GELU is rendered as below:

$$ \text{GELU}(x) = 0.5 * x * (1 + \text{Tanh}(\sqrt(2 / \pi) * (x + 0.044715 * x^3))) $$

where the square root of which part is confusing. 

I double-checked the formula, which should be:

$$ \text{GELU}(x) = 0.5 * x * (1 + \text{Tanh}(\sqrt{2 / \pi} * (x + 0.044715 * x^3))) $$

where round brackets in resource code should be brace brackets.

> _formula in [original paper](https://arxiv.org/abs/1606.08415)_
> Snipaste_2023-06-03_00-43-49

### CODE REVIEW

File: torch/nn/functional.py [hunk 1]
```diff
@@ -1734,7 +1734,7 @@ def rrelu(
 When the approximate argument is 'tanh', Gelu is estimated with
 
 .. math::
-    \text{GELU}(x) = 0.5 * x * (1 + \text{Tanh}(\sqrt(2 / \pi) * (x + 0.044715 * x^3)))
+    \text{GELU}(x) = 0.5 * x * (1 + \text{Tanh}{\sqrt(2 / \pi} * (x + 0.044715 * x^3)))
```
[the remark this answers is not in the document]
Comment (MysticalMusings) [reply torch/nn/functional.py:1737]: Yes. Oh, sorry for typos.

**Review by mikaylagawarecki — APPROVED** [review]
Thanks!

**Comment by zou3519** [comment]
Hi @MysticalMusings  I’m reaching out to you because you’re a new PyTorch contributor!  We’re trying to make it easier for new contributors and have written up a new set of Contributing Guidelines. Would you be interested in providing feedback on these?

They’re over [here](https://docs.google.com/document/d/1oNhUeGE-8ajsYaMpoV6ZQANQZVeKrdFanI9VMbFzOzc/edit#heading=h.nslheuxdbuc5). Concretely, we’d love it if you could run through the guidelines on the next PR you submit and give us any feedback on it!

**Pull request closed without merging**